src/main/resources/init_db.sql

```diff
@@ -0,0 +1 @@
+CREATE SCHEMA `cinema` DEFAULT CHARACTER SET utf8 ;
```

Comment:
you dont need to add script to create schema

Reply:
why?